components/form/Datepicker.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { DatePicker } from 'rsuite';
+import { useForm, Controller, useFormContext } from 'react-hook-form';
+import { ErrorMessage } from '.';
+
+const Datepicker = ({ name }) => {
```

Comment:
I tested the component but it is not yet working as expected. The value of datepicker is not properly mapped to React form Hook. Can you take another look at you implementation to confirm this.